src/stochastic_base.jl

```diff
@@ -104,3 +105,5 @@ function _check_const(op)
     end
     nothing
 end
+
+DiffEqNoiseProcess.wiener_randn!(rng::AbstractRNG,rand_vec::T) where {T<:Union{Bra,Ket,Operator}} = randn!(rng,rand_vec.data)
```

Comment:
Is this a bugfix for something?

Reply:
Yes, I had to define this method to enable SDEs to be solved on QO types. I didn't see an easy PR fix to submit to SciML so I defined it here. If this is too sketchy to you, let me know and I can do some more digging to work around this problem.